Demo/MapFilter/DemoSearchLocationDataSource.swift

```diff
@@ -85,47 +62,3 @@ class DemoSearchLocationDataSource: NSObject, SearchLocationDataSource {
         return true
     }
 }
-
-@available(iOS 9.3, *)
-extension DemoSearchLocationDataSource: MKLocalSearchCompleterDelegate {
```

Comment:
I'm struggling to see how this was being used...

I also don't understand exactly what `@available(ios 9.3, *)` means – how is it different from `@introduced`?

Reply:
Oh, it was used here, duh:

```
            let localSearchCompleter = MKLocalSearchCompleter()
            localSearchCompleter.delegate = self
```

Reply:
yes :)